warehouse/templates/packaging/detail.html

```diff
@@ -18,7 +18,15 @@
 {% set parsed = url|urlparse %}
 
 {% if name|lower == "download" %}
-   {% set icon = "fa fa-cloud-download-alt" %}
+   {% set icon = "fas fa-cloud-download-alt" %}
+{% elif name|lower in ["home", "homepage", "home page"] %}
+  {% set icon = "fas fa-home" %}
+{% elif name.lower().startswith(("docs", "documentation")) or parsed.netloc in ["readthedocs.io", "readthedocs.org", "rtfd.io", "rtfd.org"] or parsed.netloc.endswith((".readthedocs.io", ".readthedocs.org", ".rtfd.io", ".rtfd.org")) or parsed.netloc.startswith(("docs.", "documentation.")) %}
+  {% set icon = "fas fa-book" %}
+{% elif name.lower().startswith(("bug", "issue", "tracker", "report")) %}
+  {% set icon = "fas fa-bug" %}
+{% elif name.lower().startswith(("funding", "donate", "donation")) %}
+  {% set icon = "fas fa-coins" %}
```

Comment:
coins look a bit too much like the generic "database" icon, I think I'd personally prefer [`fas fa-donate`](https://fontawesome.com/icons/donate?style=solid)

Reply:
Of course!